allocate.go

```diff
@@ -363,6 +395,22 @@ func DisableGPU(a *ExecAllocator) {
 	Flag("disable-gpu", true)(a)
 }
 
+// StdoutWriter is used to set an io.Writer where stdout from the browser
+// will be sent
+func StdoutWriter(writer io.Writer) ExecAllocatorOption {
+	return func(a *ExecAllocator) {
+		a.stdoutWriter = writer
+	}
+}
+
+// StderrWriter is used to set an io.Writer where stderr from the browser
+// will be sent
+func StderrWriter(writer io.Writer) ExecAllocatorOption {
```

Comment:
Are you positive that you need stdout and stderr separately? Not that it's a bad thing, but does chrome even print any stdout separate from stderr?

Also, please name the parameter `w`, not `writer.

Reply:
I'm not totally certain about regular Chrome/Chromium to be honest. I know in my particular use case (custom browser), I do need stdout, but I haven't seen anything from regular Chrome being printed to stdout - it seems to be all stderr.